.travis.yml

```diff
@@ -42,8 +43,8 @@ matrix:
           env:
               - CXX_COMPILER=g++-8
               - C_COMPILER=gcc-8
-              - QT_VERSION=571
-              - QT_PKG=57
+              - QT_VERSION=58
```

Comment:
I thought there was a requirement on Qt 5.12 now?  Did you manage to make it work on the older versions?

Reply:
It works with at least 5.10, that is used for the AppImage. I haven't verified 5.8 specifically.

Reply:
(That's why I have a custom delegate chooser implementation instead of using the Qt 5.12 one)